src/app/ubs/ubs/components/ubs-personal-information/ubs-order-address/ubs-order-address.component.ts

```
@@ -188,6 +190,7 @@ export class UbsOrderAddressComponent implements OnInit, OnDestroy {
 
   deleteAddress(address: Address): void {
     this.store.dispatch(DeleteAddress({ address }));
+    this.addresses = this.addresses.filter((address) => address.id !== address.id);
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)